Py_FS/wrapper/nature_inspired/ALO.py

```diff
@@ -0,0 +1,342 @@
+"""
+Programmer: Rishav Pramanik
+Date of Development: 20/08/2021
+Antlion Optimizer
+"""
+
+from abc import abstractmethod
+import copy
+import numpy as np
+import time
+import matplotlib.pyplot as plt
+from sklearn.datasets import fetch_openml
+from sklearn.model_selection import train_test_split
+from sklearn import datasets
+from sklearn.preprocessing import LabelEncoder
+from _utilities_test import Solution, Data, compute_accuracy, compute_fitness, initialize, sort_agents, display, call_counter
+from wrapper.nature_inspired._transfer_functions import get_trans_function
+
+class ALO():
```

Comment:
Is ALO not inheriting the Algorithm class?

Reply:
Done with Changes